app/views/content/trainees/start-your-teacher-training/_content.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class="row">
+  <section class="col col-720 col-space-l-top">
+    <div class="inset">
+      <p>Your initial teacher training (ITT) will take you on an incredible journey into the classroom.</p>
+      <p>Exactly what you do during ITT will depend on the course you're on and the qualifications you're working towards. But all trainees get around 120 days of classroom experience, so you'll get to find out what life in the classroom is really like.</p>
+      <p>You'll train to teach in at least 2 schools, usually for a minimum of 6 weeks each, so you'll have plenty of time to meet other teachers and become part of supportive school communities.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Developing pupils</h2>
+    <div class="inset">
+      <p>During training, you'll get to find out what it takes to become the kind of teacher that pupils will remember years after leaving school.</p>
+      <p>Your lessons will be unobserved and observed, giving you plenty of chances to see pupils experience 'aha' moments, when your teaching helps them learn something new.</p>
+      <p>You'll be given the tools needed to inspire, motivate and challenge pupils. You'll find out how to adapt your teaching style to suit the strengths and needs of all pupils, and how to manage behaviour when things go off script.</p>
```

Comment:
```suggestion
      <p>You'll be given the tools needed to inspire, motivate and challenge pupils. You'll find out how to adapt your teaching style to suit the strengths and needs of all pupils, and how to manage behaviour and create a positive learning environment.</p>
```
@Sarah-DfE Not sure about the phrase `when things go off script`? Happy for you to change my suggestion though